lib/core/viewmodels/emergency_viewmodel.dart

```diff
@@ -0,0 +1,15 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+import 'package:notredame/core/viewmodels/security_viewmodel.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class EmergencyViewModel extends SecurityViewModel {
+  Future loadHtmlFromAssets(
```

Comment:
Please add a little documentation to explain what the function does exactly, the potential error, etc...

Reply:
added a little description